docs/development/translations.rst

```diff
@@ -0,0 +1,449 @@
+
+As explained above, the text content ``recommend-tor`` of the ``id``
+attribute in the ``<a>`` element should not be translated. Neither
+should the ``{url}`` placeholder of ``href`` attribute. Only the text
+content of the ``title`` attribute (``"How to install Tor Browser"``)
+should be translated.
+
+Other attributes
+""""""""""""""""
+
+No attribute other than ``alt`` and ``title`` should be translated.
+
+In particular, please make sure the attributes ``class``, ``id``,
+``height``, ``href``, ``rel``, ``src`` and ``width``
+are never translated.
+
+.. _adding_a_new_language:
+
+Can new languages be added to SecureDrop?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We love seeing SecureDrop translated into new languages. Just ask us
+to add yours by posting in the `translation category of the SecureDrop
+forum`_.
+
+.. _suggesting_changes_to_source_strings:
+
+Suggesting changes to source strings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you notice errors in our :ref:`source strings <weblate_glossary_source_string>`, or catch us using English
+idioms that are hard to translate, please add comments letting us
+know. At the beginning of every translation cycle in our release
+schedule, we have a few days for incorporating your feedback, and very
+much appreciate it.
+
+- :ref:`How to add a comment on a string on Weblate <how_to_add_a_comment_on_a_string_on_weblate>`
+
+.. _changing_an_existing_translation:
+
+Changing an existing translation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you think a translation can be improved, please don't run roughshod
+over another translator's work. Make a suggestion or comment first, to
+allow discussion before saving your changes.
+
+Exceptions to this policy would be:
+
+- Obvious errors in spelling, grammar, or punctuation.
+
+- A string in our interface that is supposed to match another
+  project. For example, we include instructions for adjusting Tor
+  Browser settings, so if our wording is out of date, it has to be
+  corrected to reduce confusion for people using SecureDrop.
+
+In those cases, please feel free to correct the existing translation.
+
+.. _using_language_glossaries_on_weblate:
+
+Using language glossaries on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Weblate contains an internal glossary for each language, to which
+we can add suggested translations. If a :ref:`source string <weblate_glossary_source_string>` contains terms
+from this glossary, the glossary entries will be displayed in a box on
+the right side of the translation page.
+
+|Weblate glossary sidebar screenshot|
+
+- :ref:`How to translate a phrase on Weblate <how_to_translate_a_phrase_weblate>`
+
+If you find that a *source string* contains terms from the
+:doc:`SecureDrop glossary <../glossary>` or the `EFF Surveillance
+Self-Defense glossary`_, but the glossary sidebar says ``No related
+strings found in the glossary.``, we'd really appreciate it if you
```

Comment:
Oh, syntactic coloration stops here. I don't think there is a syntax error, maybe the file size is getting beyond what GitHub colors in diffs.

Reply:
My guess is that GitHub's syntax highlighting is confused by the linebreak in the preformatted text markup -- if you ensure that it's all in one line, it may work as expected.

Reply:
You were right, it was the line break in the preformatted text!